packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -289,6 +289,7 @@ class SnackBar extends StatefulWidget {
     this.showCloseIcon,
     this.closeIconColor,
     this.duration = _snackBarDisplayDuration,
+    this.persist = false,
```

Comment:
What if developer wants it to not persist, but talkback is on? I think this may need to be nullable and if it is null, we can derive it from either action not null or accessibleNavigation is on. If it is set, we should respect it no matter what.